### PR TITLE
Add rewrites on content to use the new widgets domain

### PR DESCRIPTION
**Pull request by andrecardoso**

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(1) Chrome OS Player uses Viewer subdomain](https://trello.com/c/mr2DAawY/4062-1-chrome-os-player-uses-viewer-subdomain)

File: src/content-loader.js [hunk 1]
```diff
@@ -9,6 +9,12 @@ function fetchContent() {
     const bucketName = 'risevision-display-notifications';
     const filePath = `${displayId}/content.json`;
     return gcsClient.fetchJson(bucketName, filePath);
+  })
+  .then((contentData) => {
+    const regex = new RegExp('http(?:s?)://s3.amazonaws.com/widget-(image|video)', 'g');
+    const rewriteUrl = (match, widgetName) => `http://widgets.risevision.com/widget-${widgetName}`; // eslint-disable-line func-style
```
Comment (stulees) [inline src/content-loader.js]: @andrecardoso Do we plan on changing how we load Viewer from http to https at some point? Will we eventually go https for everything?

Comment (andrecardoso) [reply src/content-loader.js]: I wish I could do that, but there were other issues that I don't recall the details now when I tried. We might try again later and see if we can fix the issues.

Comment (stulees) [reply src/content-loader.js]: Okay thanks

**Pull request merged**